content/docs/reference/identity-provider-settings.mdx

```diff
@@ -195,7 +209,17 @@ IDP_PROVIDER=github
 
 **Identity Provider Request Params** lists the parameters you want to include as part of a sign-in request using the OAuth 2.0 code flow.
 
-Downstream application headers will be overwritten by Pomerium's headers on conflict.
```

Comment:
I don't understand what this means, and I'm wondering if we should remove it. (What's a "downstream application" in the context of an identity provider request?)

Reply:
I _believe_ downstream _request_ headers will be overwritten on conflict? I guess I'm not sure either why it's relevant.